### PR TITLE
docs: add note about use of user and logname environment variables for current username

### PR DESCRIPTION
**Pull request by albertony**
Minor doc fix, related to issue https://github.com/rclone/rclone/issues/5407. Title says it all, really.

#### Checklist

- [X] I have read the [contribution guidelines](https://github.com/rclone/rclone/blob/master/CONTRIBUTING.md#submitting-a-pull-request).
- [ ] ~I have added tests for all changes in this PR if appropriate.~
- [X] I have added documentation for the changes if appropriate.
- [X] All commit messages are in [house style](https://github.com/rclone/rclone/blob/master/CONTRIBUTING.md#commit-messages).
- [X] I'm done, this Pull Request is ready for review :-)

### CODE REVIEW

File: docs/content/tardigrade.md [hunk 1]
```diff
@@ -314,4 +314,4 @@ See [rclone about](https://rclone.org/commands/rclone_about/)
...
 
-To fix these, please raise your system limits. You can do this issuing a `ulimit -n 65536` just before you run rclone, inside a bash script or inside the `$USER/.bashrc` file. If you need to change limits system-wide, this usually takes place at `/etc/sysctl.conf` and/or `/etc/security/limits.conf` on Linux, but please refer to your operating system manual.
+To fix these, please raise your system limits. You can do this issuing a `ulimit -n 65536` just before you run rclone, inside a bash script or inside the `$HOME/.bashrc` file. If you need to change limits system-wide, this usually takes place at `/etc/sysctl.conf` and/or `/etc/security/limits.conf` on Linux, but please refer to your operating system manual.
```
Comment (raphaelcohn) [inline docs/content/tardigrade.md:317]: Actually, this is more complex than simply `.bashrc`; it depends on whether a terminal is interactive or not (eg `.profile`). The rules are different for the ash shell (the default in some Ubuntu derivatives) and the busybox variant of ash (eg Alpine Linux). Probably best to leave it at hints and walk away!

Comment (albertony) [reply docs/content/tardigrade.md:317]: Oh no.. I just wanted to fix the path `$USER/.bashrc` to `$HOME/.bashrc`, which I assumed was the intention. 😭 But thanks for the input! 😄
Not my original text, so not sure how much I want to wreck it at this point.. perhaps I can try to insert that this is an example for interactive bash or something.. :thinking:
Thanks again.

Comment (raphaelcohn) [reply docs/content/tardigrade.md]: No problem. Just mention ulimits and drop any shell stuff.

**Comment by ncw** [comment]
Thank you - this is a nice fix :-)

I will merge this now.

**Pull request merged**